docs/src/pages/components/masonry/ImageMasonry.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function ImageMasonry() {
+  return (
+    <Masonry
+      columns={{ xs: 3, sm: 5, md: 7 }}
+      spacing={{ xs: 1, sm: 2, md: 3 }}
+      sx={{ height: 300 }}
+    >
+      {itemData.map((item) => (
+        <MasonryItem key={item.img}>
+          <img
+            srcSet={`${item.img}?fit=crop&auto=format 1x,
```

Comment:
I don't think we need `fit=crop` here. Perhaps we should limit the size of image being retrieved to improve the load time.